app/src/main/java/com/calculator/MainActivity.java

```diff
@@ -1,16 +1,261 @@
 package com.calculator;
 
+import androidx.annotation.Nullable;
 import androidx.appcompat.app.AppCompatActivity;
 
+import android.content.Intent;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.Switch;
+import android.widget.TextView;
+
+import com.calculator.utilities.Utility;
 
 public class MainActivity extends AppCompatActivity {
+    Calculator calculator = new Calculator();
```

Comment:
almost all creation of instances should be placed inside the onResume method because if collector garbage remove the object, the object should be restored in order to avoid errors, so please move this line to onResume method or enclosing this in a Singleton object and before use, it asks for the current object state if is null or not